### PR TITLE
new md

### PR DESCRIPTION
**Pull request by Yoshyyy**
new md

### CODE REVIEW

**Comment by Yoshyyy** [comment]
cheers man

**Comment by Yoshyyy** [comment]
ive done this, do i need to put your master.md back to what it was?

File: Yoshyyy/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+สวัสดีค่ะ
+
+ขอบคุณสำหรับ workshop เรื่อง git มากๆ เลยค่ะ :) ขอแชร์บทความนี้ให้กับเพื่อนๆ ที่ยังไม่เคยใช้ git นะคะ
```
Comment (Phonbopit) [inline Yoshyyy/README.md]: ยินดีครับ

Comment (Yoshyyy) [reply Yoshyyy/README.md]: hahaha, should it say something else?

**Comment by Phonbopit** [comment]
Nice 👍

**Pull request merged**